src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/FeatureLayerRenderingModeMap/FeatureLayerRenderingModeMap.xaml.cs

```diff
@@ -37,6 +37,10 @@ public FeatureLayerRenderingModeMap()
 
         private void Initialize()
         {
+            // Create the maps.
+            MyStaticMapView.Map = new Map();
```

Comment:
How about setting InitialViewpoint here, like on other platforms? This UWP sample currently starts off zoomed waaay out.